buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -264,7 +267,10 @@ message GetCommitNodesRequest {
     bool allow_paths_not_exist = 3;
   }
   // The File sets to request.
-  repeated Value values = 1 [(buf.validate.field).repeated.min_items = 1];
+  repeated Value values = 1 [
```

Comment:
This isn't a ref, but I presume we want to set an upper limit for these repeated values as well?

Reply:
Yea